src/ucp/core/ucp_rkey.c

```diff
@@ -266,6 +273,12 @@ ucs_status_t ucp_memh_pack(ucp_mem_h memh, ucp_memh_pack_params_t *params,
     return status;
 }
 
+ucs_status_t ucp_memh_pack(ucp_mem_h memh, ucp_memh_pack_params_t *params,
+                           void **buffer_p, size_t *buffer_size_p)
+{
+    return ucp_memh_pack_internal(memh, params, 0, buffer_p, buffer_size_p);
```

Comment:
I guess this always returns UCS_ERR_UNSUPPORTED now?

Reply:
yes, it returns `UCS_ERR_UNSUPPORTED`, because new rkey_buffer format wasn't implemented